smugmug.com.custom-domain.json

```diff
@@ -1,21 +1,23 @@
-{  
-   "providerId":"smugmug.com",
-   "providerName":"SmugMug",
-   "serviceId":"custom-domain",
-   "serviceName":"SmugMug Custom Domain",
-   "version": 2,
-   "logoUrl":"https://www.domainconnect.org/wp-content/uploads/2016/02/smugmug-300x80.png",
-   "description":"Enables a domain to work with SmugMug",
-   "records":[
-      {  
+{
+   "providerId": "smugmug.com",
+   "providerName": "SmugMug",
+   "serviceId": "custom-domain",
+   "serviceName": "SmugMug Custom Domain",
+   "version": 3,
+   "logoUrl": "https://cdn.smugmug.com/img/domain-connect/smugmug-logo.png",
+   "description": "Enables a domain to work with SmugMug",
+   "syncRedirectDomain": "smugmug.com, smugmug.net",
+   "records": [
+      {
          "type": "REDIR301",
-         "target": "www.%fqdn%"
+         "target": "www.%domain%",
```

Comment:
I also should have noticed this one during the first review.
HTTP Redirect target shall be defined as an URL, so just a domain name would not work.

Reply:
so you're saying scheme is required?  ie. `https://www.%domain%`. (we provide certs so hard-coding https is fine)

Reply:
correct